jobs/azure-discovery/src/main/java/com/tmobile/pacbot/azure/inventory/file/AssetFileGenerator.java

```diff
@@ -689,54 +740,62 @@ public void generateFiles(List<SubscriptionVH> subscriptions, String filePath) {
 					return;
 				}
 				try {
-
+					targetTypeList.add("subscription");
 					FileManager
 							.generateSubscriptionFiles(subscriptionInventoryCollector.fetchSubscriptions(subscription));
 					log.info("subscription data saved!");
+					targetTypeList.remove("subscription");
 				} catch (Exception e) {
 					e.printStackTrace();
 					Util.eCount.getAndIncrement();
 				}
 			});
+
 			executor.execute(() -> {
 				if (!(isTypeInScope("functionapp"))) {
 					log.info("no target type found for functionApp!!");
 					return;
 				}
 				try {
+					targetTypeList.add("functionapp");
 					FileManager
 							.generateFunctionAppFiles(functionAppInventoryCollector.fetchFunctionAppDetails(subscription));
 					log.info("subscription data saved!");
```

Comment:
let's fix it - it's not a subscription

Reply:
also if we plan to keep this logs here (me personally I don't any help from them here) - replace target type with constants you will be creating (see one of my other comments)